torchgeo/datasets/etci2021.py

```diff
@@ -274,14 +274,17 @@ def plot(
         """
         vv = np.rollaxis(sample["image"][:3].numpy(), 0, 3)
         vh = np.rollaxis(sample["image"][3:].numpy(), 0, 3)
-        water_mask = sample["mask"][0].numpy()
+        mask = sample["mask"].squeeze(0)
```

Comment:
There was a bug in this plotting function. Basically, `dataset[idx]` returns a 1xHxW tensor for "test" and 2xHxW for "train/val", so the previous code worked fine for this. But our datamodule returns a HxW tensor and this didn't work properly.

Reply:
@calebrob6 this is the other ETCI 2021 bug